infra/airflow/Dockerfile

```diff
@@ -12,8 +12,8 @@ COPY requirements.txt /opt/airflow
 
 # working in the /opt/airflow
 WORKDIR /opt/airflow
-RUN wget -c https://github.com/RMLio/rmlmapper-java/releases/download/v5.0.0/rmlmapper-5.0.0-r362-all.jar -P .rmlmapper/
-RUN mv .rmlmapper/rmlmapper-5.0.0-r362-all.jar .rmlmapper/rmlmapper.jar 2>/dev/null
+RUN mkdir -p ./.rmlmapper
```

Comment:
Why not just mkdir -p .rmlmapper

Reply:
asa a pus Stefan la artefacts, ca sa fie 100% sigur